checker/votour.ml

```diff
@@ -261,6 +262,7 @@ let pop () = match !stk with
   | i::s -> stk := s; i
   | _ -> failwith "empty stack"
 
+[@@@ocaml.warning "-52"]
```

Comment:
Instead of disabling this warning please fix it by replacing the `failwith "empty_stack"` and `failwith "forbidden"` with specific exceptions.